compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/matchers/IrFunctionMatcher.kt

```diff
@@ -97,3 +99,13 @@ internal fun IrFunctionMatcherContainer.singleArgumentExtension(
     fqName { it == fqName }
     return this
 }
+
+internal fun IrFunctionMatcherContainer.singleArgumentExtensionBySimpleType(
```

Comment:
This looks suspicious, why not change `progressionClassesTypes` to contain IR types?

Reply:
Ah, already done, sorry.

Reply:
Oops, fixed.